src/components/ErrorTest/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './ErrorTest.module.scss';
+
+const { errorTest__button } = styles;
+
+type ErrorTestProps = Record<string, never>;
+type ErrorTestState = {
+  throwError: boolean;
+};
```

Comment:
I suggest creating a separate file within this folder and moving the types into this file